README.md

```diff
@@ -78,10 +78,10 @@ You will need to have an account on Nextflow Tower (see [Plans and pricing](http
    twkit hello-world-config.yml
    ```
 
-   <b>Note</b>: The Tower CLI expects to connect to a Tower instance that is secured by a TLS certificate. If your Tower instance does not present a certificate, you will need to qualify and run your `tw` commands with the `--insecure` flag. For example:
+   <b>Note</b>: The Tower CLI expects to connect to a Tower instance that is secured by a TLS certificate. If your Tower instance does not present a certificate, you will need to qualify and run your `tw` commands with `--cli` followed by the `--insecure` flag. For example:
```

Comment:
@adamrtalbot I think this is quite ugly to have to provide any additional `tw` cli options after `--cli` to twkit so it can parse these correctly to the `_tw_run()` method  in the `Tower()` class and not confuse it with `twkit` specific arguments such as `--delete` and `--dryrun` but I am not sure how else to handle this. Any suggestions?